### DISCUSSION TITLE
Disable guest

### DISCUSSION THREAD

**Discussion by francodimasi** [pretix/pretix#3107] [Q&A] [answered]
Hi all, is there any way to disable Guests?

I need to force users to create customer accounts in order to purchase tickets.

Thanks for your support!

We Love Pretix!

Franco.

**Comment by raphaelm** [comment, marked as answer]
No, there currently isn't. Why do you need that?

**Reply by francodimasi** [reply]
Hi Rapha, my pleasure.

There is a lot a fraud in Argentina regarding credit card purchases and payment gateways usually ask for a better KYC to merchants. We want to create a stronger signup process with photo ID. That's why.

any ideas or workarrounds?

Thanks, Franco.

**Reply by pc-coholic** [reply]
(As always: not a lawyer, not legally binding information)

Quick interjection to the 

> payment gateways usually ask for a better KYC to merchants.

Please carefully verify the Merchant Agreement that apply to you. In most instances, card networks such as VISA and/or Mastercard allow the merchant to request identification transactions. But a merchant *cannot* refuse the the transaction if the customer does not present identification or refuses to do so. Refusing to accept a card because the customer didn't show you their ID is grounds for losing your merchant acceptance agreement with a card network.

If you are only dealing through a payment gateway and they ask you to request customer ID card copies, your payment gateway might be "out of whack"... There are established ways to authenticate the card holder besides having them only enter the PAN and CVC: "Verified by Visa" and "Mastercard Secure Code" just to name a few. You/your payment provider should be able to force those validations - even on transactions that would normally be "2 factor optional" in order to combat fraud.

**Reply by francodimasi** [reply]
Thanks for your answer Martin, totally agree. I will double check with my payment processor. Thanks.
